### ISSUE TITLE
Ideas/discussion around source map support in Nuxt 3

### ISSUE THREAD

**Issue by ffxsam** [nuxt/nuxt.js#14055]

**Comment by toniengelhardt** [comment]
Would also be interested in this to get source map support in Sentry without exposing them directly with the bundle.

For reference, here is how Sentry suggests integrating with Vite:
https://docs.sentry.io/platforms/javascript/guides/vue/sourcemaps/uploading/vite/

**Comment by huynhj93** [comment]
I'm also having this issue, it seems like the source maps are uploaded but Sentry doesnt recognize the uploaded debug ids.See https://github.com/getsentry/sentry-javascript/discussions/9459#discussioncomment-7530582

**Comment by OrbisK** [comment]
I don't think this is relevant anymore. The [Sentry Nuxt Integration](https://docs.sentry.io/platforms/javascript/guides/nuxt/) and some sourcemap improvements should fulfil the original idea.

```ts
export default defineNuxtConfig({
  sourcemap: { client: "hidden" },
});
```
